app/Http/Controllers/DisplayDataController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use DataTables;
+use App\User;
+use Yajra\DataTables\Services\DataTable;
```

Comment:
Nice

Reply:
Thanks, 
I heard from a recruiter that you were hiring again.

Emiliano
415-320-9390 cell